apps/dashboard/app/views/projects/import.html.erb

```diff
@@ -1,10 +1,55 @@
+<%= javascript_include_tag('projects_import', nonce: true, type: 'module') %>
+
 <div class='page-header text-center pb-3'>
   <h1>
     Import a Shared Project
   </h1>
 </div>
-# TODO: Add a new partial for this. May be similar to index with a list of cards or a simple search
-# Right now it point to "Create new Project" form
+
+<%
+  path_selector_id = "project_directory_path_selector"
+
+  path_selector_locals = {
+    path_selector_id: path_selector_id,
+    show_files: false,
+    show_hidden: false,
+    file_pattern: '',
+    initial_directory: CurrentUser.home,
+    table_id: "#{path_selector_id}_table",
+    breadcrumb_id: "#{path_selector_id}_breadcrumb",
+    button_id: "#{path_selector_id}_button",
+    input_field_id: 'project_directory',
+    favorites: false,
+  }
+%>
+
 <%= bootstrap_form_for(@project, url: project_import_save_path, method: :post) do |form| %>
-  <%= render partial: "form", locals: { form: form }%>
+<div class='card'>
+  <div class='card-group'>
+    <div class='card'>
+      <div class='card-body'>
+        <div class="col">
+          <div class="field">
+            <%= form.text_field(:directory, placeholder: I18n.t('dashboard.jobs_project_directory_placeholder'),
+                                required: false,
+                                data: { 'path-selector': true })%>
+
+            <button type="button" class="btn btn-primary mt-2" data-bs-toggle="modal" data-bs-target="#<%= path_selector_id %>">
+              <%= I18n.t('dashboard.select_path') %>
+            </button>
+
+            <%= render(partial: 'shared/path_selector_table', locals: path_selector_locals ) %>
+          </div>
+        </div>
+      </div>
+      </div>
+    </div>
+  </div>
+  <br>
+  <p>
+    <%= form.submit 'Save', class: 'btn btn-primary', title: 'Save project' %>
+    <%= form.button 'Reset', type: :reset, class: 'btn btn-default', title: 'Clear form fields' %>
+    <%= link_to 'Back', projects_path, class: 'btn btn-default', title: 'Return to projects page' %>
```

Comment:
We should likely internationalize these. I think we have Save, not sure about Reset or Back.

Reply:
Done